backend/src/main/java/com/github/redstylzz/backend/filter/JwtAuthFilter.java

```diff
@@ -39,16 +39,7 @@ protected void doFilterInternal(HttpServletRequest request, HttpServletResponse
         String token = getToken(request);
 
         LOG.debug("JWTAuthFilter Token: " + token);
```

Comment:
I wouldn't Log the token even if you have a short expiration time, same as the password